### ISSUE TITLE
Remove a Workspace

### ISSUE THREAD

**Issue by Scribilicious** [notaapp/nota#106]
Maybe the question is already answered. How can I remove a workspace from my workspaces?

**Comment by astoilkov** [comment]
Yes. You can look at the following issue: https://github.com/notaapp/nota/issues/73.

If you have any additional questions you can ask them there. I am closing this issue as it's duplicate and it's hard to manage duplicates.
